ibm_catalog.json

```diff
@@ -71,22 +71,23 @@
             },
```

Comment:
for `Resource group only` iam permission, role_crn should have one indent less

<img width="1353" alt="image" src="https://github.com/user-attachments/assets/8ccdf1e7-72bf-4e53-b6e2-780f981e3a3b" />

Reply:
check security-enforced as well